cura/SettingOverrideDecorator.py

```diff
@@ -51,8 +51,8 @@ def __deepcopy__(self, memo):
     def getActiveExtruder(self):
         return self._extruder_stack
 
-    def _onSettingChanged(self, instance, property):
-        if property == "value":  # Only reslice if the value has changed.
+    def _onSettingChanged(self, instance, attribute): # Reminder: 'property' is a built-in function
+        if attribute == "value":  # Only reslice if the value has changed.
```

Comment:
I would prefer ```property_name``` over ```attribute```. The word "property" is fairly embedded in the settings architecture (a setting has several properties such as type, value, min_value, etc), so using a synonym does not help in the clarity of what is being passed here. Good catch though.